src/Orc.Snapshots/Extensions/ZipEntryExtensions.cs

```diff
@@ -8,18 +8,31 @@
 namespace Orc.Snapshots
 {
     using System.IO;
-    using Ionic.Zip;
+    using System.IO.Compression;
 
     internal static class ZipEntryExtensions
     {
-        public static byte[] GetBytes(this ZipEntry entry)
+        public static byte[] GetBytes(this ZipArchiveEntry entry)
         {
-            using (var memoryStream = new MemoryStream())
+            using (var stream = entry.Open())
             {
-                entry.Extract(memoryStream);
+                using (var memoryStream = new MemoryStream())
+                {
+                    stream.CopyTo(memoryStream);
+                    return memoryStream.ToArray();
+                }
+            }
+        }
 
-                return memoryStream.ToArray();
+        public static void OpenAndWrite(this ZipArchiveEntry entry, byte[] bytes)
+        {
+            using (var stream = entry.Open())
+            {
+                using (var streamWriter = new StreamWriter(stream))
+                {
+                    streamWriter.Write(bytes);
```

Comment:
I have a feeling after entry.Open() stream contains the previous content of entry
It means you need somehow cleanup it before write e.g. use `SetLength` or remove entry and add a new one

Reply:
Didn't notice mixing with previous content here